src/modules/addComment.js

```diff
@@ -0,0 +1,33 @@
+import displayComments from './displayComments.js';
+
+const addComment1 = async (comObj, newid) => {
+  const getApiComment = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/eHLzmWWPvFRWRTA0brmG/comments';
+  await fetch(getApiComment, {
```

Comment:
Your API is working as expected.  :tada: I suggest you hide your API key for security purposes. Use dotenv to do so :point_right: [Click me](https://www.npmjs.com/package/dotenv)